packages/block-editor/src/components/block-list/block.js

```diff
@@ -161,6 +164,10 @@ function BlockListBlock( {
 		!! wrapperProps[ 'data-align' ] &&
 		! themeSupportsLayout;
 
+	// Support for sticky position in classic themes with alignment wrappers.
+
+	const isSticky = className?.includes( 'is-position-sticky' );
```

Comment:
Non-blocking question: should we factor in potential future position values by checking for `is-position-` rather than `is-position-sticky`, or should we leave that 'til we have a concrete plan in place for extending supported position types?

Reply:
Great question! Other position types will likely have different problems: with either absolute or fixed, we'll have to deal with wide/full width no longer working for the positioned elements. If we do go down that path it's possible that positioning a block means disabling wide/full on it, so I think it's fine to make this explicitly support sticky for now and reevaluate as needed 😄

Reply:
Sounds good, it also means we're playing it safe by only having explicit handling for one feature we specifically know about 👍